src/api/app/views/layouts/webui/responsive_ux/_places.html.haml

```diff
@@ -1,13 +1,13 @@
-.navbar-collapse.offcanvas-collapse.navbar-dark
-  .navbar-nav
-    .nav.justify-content-end.py-2
-      %button.navbar-toggler{ 'type': 'button', data: { toggle: 'offcanvas' }, aria: { expanded: 'false', label: 'Toggle navigation' } }
-        %i.fas.fa-times
-  %ul.navbar-nav.ml-auto.pt-0.text-nowrap.menu-options
-    %li.nav-item
-      = link_to(user_path(User.session!), id: 'link-to-user-home', class: 'nav-link') do
-        = image_tag_for(User.session, size: 23, custom_class: 'rounded-circle bg-light mr-2')
-        Profile
+%h5.pt-2.text-light{ class: navigation == :left ? 'pl-2' : '' }
```

Comment:
```suggestion
%h5.pt-4.text-light{ class: navigation == :left ? 'pl-2' : '' }
```

Reply:
A little more padding would create a more clear seperation between the places and actions

Reply:
I agree with you. I've applied your suggestion.